services/src/northbound-service/northbound/src/main/java/org/openkilda/northbound/service/impl/FlowServiceImpl.java

```diff
@@ -168,9 +169,9 @@
         try {
             payload = new FlowCreateRequest(new FlowDto(input), input.getDiverseFlowId());
         } catch (IllegalArgumentException e) {
-            logger.error("Can not parse arguments: {}", e.getMessage());
+            logger.error(format("Can not parse arguments: %s", e.getMessage()), e);
```

Comment:
Why do you use this way of composing a message?

Reply:
fixed